packages/twenty-zapier/authentication.js

```diff
@@ -0,0 +1,48 @@
+const test = async (z, bundle) => {
```

Comment:
is it possible to write the whole zapier app in typescript?

Reply:
well, zapier provides [a cli](https://platform.zapier.com/quickstart/cli-tutorial) which generates `.js` files. There is some [workarounds to use typescript](https://medium.com/@Stefan1129/using-typescript-interfaces-within-your-zapier-app-c90077aede3d) but I am not sure it worths it

Reply:
Would be great to use typescript https://github.com/zapier/zapier-platform/tree/main/example-apps/typescript